src/Tools/dotnet-counters/ConsoleWriter.cs

```diff
@@ -37,6 +38,7 @@ public ConsoleWriter()
             displayLength = new Dictionary<string, int>();
             knownProvidersRowNum = new Dictionary<string, int>();
             unknownProvidersRowNum = new Dictionary<string, int>();
+            leftAlign = 45;
```

Comment:
45 is?

Reply:
the initial width of the counter names column, so that's where the counter values would start

Reply:
I'd suggest leaving a comment on why this is 45, or even better, programmatically calculating this by finding the maximum length of the "known" counter names so that when a counter with a longer name is added in the future, this value changes with it.

Reply:
Also you can assign this variable in the declaration above (line 20)

Reply:
Agree with Sung here, computing this programmatically would be preferred.